Esri/arcgis-maps-sdk-kotlin-samples
# Show geodesic path between two points

## Description
PR to add a new Kotlin sample _"Show geodesic path between two points"_ in `Analysis` category. This sample calculates a geodesic path between two points and measures its distance.

## Links and Data
Sample Epic: `runtime/kotlin/issues/1806`

## What To Review
-  Review the code to make sure it is easy to follow like other samples on Android
- `README.md` and `README.metadata.json` files

## How to Test
Run the sample on the sample viewer or the repo.
Tap anywhere on the map once loaded. A geodesic path is drawn and the bottom Textview should display the distance between the points in kilometers.
